assets/js/modules/analytics/components/common/GA4PropertyNotice.js

```diff
@@ -32,18 +32,19 @@ import { __ } from '@wordpress/i18n';
 import SettingsNotice, { TYPE_INFO } from '../../../../components/SettingsNotice';
 import Link from '../../../../components/Link';
 
+const LearnMore = () => (
+	<Link
+		href="https://sitekit.withgoogle.com/documentation/ga4-analytics-property/"
+		external
+		inherit
+	>
+		{ __( 'Learn more here.', 'google-site-kit' ) }
+	</Link>
+);
```

Comment:
Why @eugene-manuilov ? Because there's a Site Kit rule of one-component-per-file?  I copied this code from elsewhere so that's not the case :smile: 

Happy to make a ticket to enforce this in https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-multi-comp.md

Otherwise it feels random what I should change :joy: